pennylane_lightning/core/src/simulators/lightning_kokkos/StateVectorKokkos.hpp

```diff
@@ -862,74 +863,29 @@ class StateVectorKokkos final
     std::mutex init_mutex_;
     std::unique_ptr<KokkosVector> data_;
     inline static bool is_exit_reg_ = false;
-    // clang-format off
+
     /**
-    * @brief Register gate operations in the gates_indices_ attribute:
-    *        an unordered_map mapping strings to GateOperation enumeration keywords.
-    */
+     * @brief Register gate operations in the gates_indices_ attribute:
+     *        an unordered_map mapping strings to GateOperation enumeration
+     * keywords.
+     */
     void init_gates_indices_() {
```

Comment:
Could you make this constexpr?

Reply:
Thought about that too. I figured it's either a bit of time, or a bit of memory, and I would get your opinions. If @AmintorDusko agrees, we could define once and for all this reversed map in `Constant.hpp` along with all such definitions.

Reply:
I'm OK with that.

Reply:
Ok, let me see what I can do.

Reply:
I think [this](https://stackoverflow.com/a/8026952) is the issue here.

Reply:
yeah, not with using map! let's merge this PR and we can get back to this later :)